helpers/QtiPackageExporter.php

```diff
@@ -20,40 +20,99 @@
 
 namespace oat\taoQtiTest\helpers;
 
-use League\Flysystem\Util;
-use oat\oatbox\service\ConfigurableService;
+use common_Exception;
+use common_exception_Error;
+use common_report_Report;
+use oat\oatbox\filesystem\File;
+use oat\oatbox\filesystem\FileSystemService;
+use oat\tao\helpers\FileHelperService;
+use oat\tao\model\service\InjectionAwareService;
 use taoQtiTest_models_classes_export_TestExport22 as TestExporter;
-use core_kernel_classes_Resource as RdfResource;
-use tao_helpers_File as FileHelper;
 
-class QtiPackageExporter extends ConfigurableService
+class QtiPackageExporter extends InjectionAwareService
```

Comment:
InjectionAwareService: It's good to know about it.

Reply:
Unfortunately it requires custom script to register a class if your dependencies are something  bigger than scalar or simple class `new DependencyClass()`